contributing.md

```diff
@@ -162,10 +162,20 @@ get acquainted with this development process.
 
 1. **Choose a base branch.**
    - If your changes will break API or ABI, then base your new branch off of `main`.
+     If this package's `main` branch is not part of the upcoming Gazebo collection,
+     create an issue on the package repository asking its `main` branch to be added
+     to the next collection.
+     For example, at the time this documentation is being written the upcoming Jetty
+     collection is [using the gz-cmake4 branch](https://github.com/gazebo-tooling/gazebodistro/blob/bc0de6a86d6ab84106f208395969353d43b18dac/collection-jetty.yaml#L6)
+     rather than `main`.
```

Comment:
```suggestion
     For example, at the time this documentation is being written the upcoming Jetty
     collection is [using the gz-cmake4 branch](https://github.com/gazebo-tooling/gazebodistro/blob/bc0de6a86d6ab84106f208395969353d43b18dac/collection-jetty.yaml#L6)
     rather than `main`, which means gz-cmake's `main` branch is actually not part
     of a collection yet.
```

Reply:
thanks! I applied your suggestion